sqlmesh/core/model/definition.py

```diff
@@ -533,7 +533,7 @@ def update_schema(
             if mapping_schema:
                 nested_set(
                     self.mapping_schema,
-                    tuple(str(part) for part in table.parts),
+                    tuple(part.sql(copy=False)[1: -1] for part in table.parts),
```

Comment:
Is it guaranteed we'll always have quotes here?

Reply:
yes, because all tables are fqn quoted